components/drawer/style/drawer.less

```diff
@@ -191,8 +191,7 @@
     }
 
     &-close-only {
-      padding-bottom: 0;
-      border: none;
+      justify-content: flex-end;
```

Comment:
Do not change this stylesheet because it is automatically updated by our ng-zorro-bot based on the ant-design's styles.

Reply:
Has restore